web/src/services/api-register.ts

```diff
@@ -27,6 +27,7 @@ export default {
   admin_user_logout: `/api/${version}/logout`,
   udx_register_device: `/usl/v1/registerDevice`,
   udx_registration_token: `/usl/v1/registrationToken`,
+  udx_saas: `/usl/v1/udx_saas`,
```

Comment:
Please use version variable instead of hardcoded v1

Reply:
Removed v1 and added version variable.